app/src/main/java/com/hamdan/forzenbook/view/login/CreateAccount.kt

```diff
@@ -161,15 +145,27 @@ private fun Content(
         imeAction = ImeAction.Next,
         onNext = { focusManager.moveFocus(FocusDirection.Down) }
     )
-    if (firstNameError && firstName.isNotEmpty()) {
-        Text(text = stringResource(R.string.create_account_first_name_error))
+    if (firstName.isNotEmpty()) {
+        when (firstNameError) {
```

Comment:
Stuff like this where you have an empty else block with nothing makes me think it makes much more sense to be an if